listenbrainz/webserver/views/api.py

```diff
@@ -116,13 +116,22 @@ def get_listens(user_name):
     for listen in listens:
         listen_data.append(listen.to_api())
 
+    latest_listen = db_conn.fetch_listens(
+        user_name,
+        limit=1,
+        to_ts=max_ts,
+    )
+    latest_listen_ts = latest_listen[0].ts_since_epoch if len(latest_listen) > 0 else 0
```

Comment:
This assumes that the user has at least one listen, it should have a guard around it, because this won't always be the case

Reply:
my bad, I didn't see if the if-check the first time. Ignore me!